sru-templates/manual/sru-vars.rc

```diff
@@ -0,0 +1,112 @@
+# This file is sourced by all manual SRU cloud tests
+# It relies on user-specific variable overrides in SRU_VARS_LOCAL_RC.
+# Set the variables accordingly for your cloud configuration and usernames
+
+PRESERVE_INSTANCE=false   # can be set as cmdline param
+UBUNTU_RELEASE=""         # The release to test specfied as cmdline param
+LP_USER="UNSET"            # specify your launchpad username
+PROPOSED_SCRIPT=setup_proposed.sh
+
+USE_DEV_PPA=0  # set to 1 to use ppa:cloud-init-dev/proposed
+
+SAMPLE_CLOUDCONFIG="sethostname.yaml"
+# Key used to interact with cloud-instances
+SSH_KEY="$HOME/.ssh/id_rsa.pub"
+
+
+AZURE_REGION="UNSET"          # specify your region name
+AZURE_VNET_NAME="UNSET"       # specify your virtual network name
+AZURE_NETWORK_SECURITY_GROUP="UNSET" # specify your network security group
+AZURE_NIC_NAME="UNSET"        # specify a preferred nic name
+AZURE_RESOURCE_GROUP="UNSET"  # specify your resource group
+AZURE_BOOT_DIAG="UNSET"       # your boot diagnostics group: bootdiag
+AZURE_ADVANCED_NIC_TESTS=true  # set false to avoid time-consuming test setup
+
+GCE_ZONE="UNSET"              # specify preferred zone: us-central1-b
+
+OPENSTACK_ADMIN_NET_ID="UNSET" # your openstack admin net on serverstack
+OPENSTACK_SSH_KEY_NAME="UNSET" # your openstack ssh key name on serverstack
+
+VMWARE_PASSWORD="UNSET"        # set to vcenter admin password
+
+# Define custom values for the above variables in ~/sru-vars-local.rc
+SRU_VARS_LOCAL_RC=$HOME/sru-vars-local.rc
```

Comment:
I know you already got comments on this, but my preference would be to rename this file as `sru-vars.rc.template`. The README can then specify to `cp sru-vars.rc.template -> sru-vars.rc` and modify as needed. `sru-vars.rc` can then be added to `.gitignore` so git doesn't complain about changes. I'd rather not have this file in my home directory.

If we do think it belongs in our home directory, can we at least prepend it with a dot so it's hidden by default?

Reply:
Thanks @TheRealFalcon I've renamed it to sru-templates/manual/sru-vars.template and documented that overrides can be created with:
```
  $ echo "# Custom SRU verification Variable overrides" > .sru-vars.rc
  $ grep '="UNSET' sru-templates/manual/sru-vars.template >> .sru-vars.rc
  # Then set appropriate values for your LP_USER and specific cloud config
  $ vi .sru-vars.rc
```